package.json

```diff
@@ -4,6 +4,7 @@
     "remark-cli": "^7.0.1",
     "remark-preset-lint-recommended": "^3.0.3",
     "remark-validate-links": "^10.0.0",
+    "sdpoker": "AMWA-TV/sdpoker",
```

Comment:
I think it makes sense to use `AMWA-TV/sdpoker` master rather than e.g. `AMWA-TV/sdpoker#v0.2.0`, right?